bot/constants.py

```diff
@@ -1,703 +1,728 @@
-    with open("config.yml", encoding="UTF-8") as f:
-        user_config = yaml.safe_load(f)
-    _recursive_update(_CONFIG_YAML, user_config)
+    # Staff
+    admins = 365960823622991872
+    admin_spam = 563594791770914816
+    defcon = 464469101889454091
+    helpers = 385474242440986624
+    incidents = 714214212200562749
+    incidents_archive = 720668923636351037
+    mod_alerts = 473092532147060736
+    mod_meta = 775412552795947058
+    mods = 305126844661760000
+    nominations = 822920136150745168
+    nomination_voting = 822853512709931008
+    organisation = 551789653284356126
 
+    # Staff announcement channels
+    admin_announcements = 749736155569848370
+    mod_announcements = 372115205867700225
+    staff_announcements = 464033278631084042
+    staff_info = 396684402404622347
+    staff_lounge = 464905259261755392
 
-def check_required_keys(keys):
-    """
-    Verifies that keys that are set to be required are present in the
-    loaded configuration.
-    """
-    for key_path in keys:
-        lookup = _CONFIG_YAML
-        try:
-            for key in key_path.split('.'):
-                lookup = lookup[key]
-                if lookup is None:
-                    raise KeyError(key)
-        except KeyError:
-            raise KeyError(
-                f"A configuration for `{key_path}` is required, but was not found. "
-                "Please set it in `config.yml` or setup an environment variable and try again."
-            )
-
-
-try:
-    required_keys = _CONFIG_YAML['config']['required_keys']
-except KeyError:
-    pass
-else:
-    check_required_keys(required_keys)
-
-
-class YAMLGetter(type):
+    # Voice Channels
+    admins_voice = 500734494840717332
+    code_help_voice_0 = 751592231726481530
+    code_help_voice_1 = 764232549840846858
+    general_voice_0 = 751591688538947646
+    general_voice_1 = 799641437645701151
+    staff_voice = 412375055910043655
+
+    black_formatter = 846434317021741086
+
+    # Voice Chat
+    code_help_chat_0 = 755154969761677312
+    code_help_chat_1 = 766330079135268884
+    staff_voice_chat = 541638762007101470
+    voice_chat_0 = 412357430186344448
+    voice_chat_1 = 799647045886541885
+
+    big_brother_logs = 468507907357409333
+    duck_pond = 637820308341915648
+    roles = 851270062434156586
+
+
+Channels = _Channels()
+
+
+class _Roles(EnvConfig):
+
+    EnvConfig.Config.env_prefix = "roles."
+
+    # Self-assignable roles, see the Subscribe cog
+    advent_of_code = 518565788744024082
+    announcements = 463658397560995840
+    lovefest = 542431903886606399
+    pyweek_announcements = 897568414044938310
+    revival_of_code = 988801794668908655
+    legacy_help_channels_access = 1074780483776417964
+
+    contributors = 295488872404484098
+    help_cooldown = 699189276025421825
+    muted = 277914926603829249
+    partners = 323426753857191936
+    python_community = 458226413825294336
+    sprinters = 758422482289426471
```

Comment:
Let's drop this. it's only used for a (now unused) whitelist. If you stumble upon any unused constants like this, feel free to drop as well.